File: src/main/java/com/raoulvdberge/refinedstorage/item/QuartzEnrichedIronItem.java
package com.raoulvdberge.refinedstorage.item;

import com.raoulvdberge.refinedstorage.RS;
import net.minecraft.item.Item;

public class QuartzEnrichedIronItem extends Item {
    public QuartzEnrichedIronItem() {
        super(new Item.Properties().group(RS.MAIN_GROUP));

        this.setRegistryName(RS.ID, "quartz_enriched_iron");
    }
}

File: src/main/kotlin/com/refinedmods/refinedstorage/item/QuartzEnrichedIronItem.kt
package com.refinedmods.refinedstorage.item

import com.refinedmods.refinedstorage.RS
import net.minecraft.item.Item

class QuartzEnrichedIronItem : Item(Properties().group(RS.MAIN_GROUP)) {
    init {
        this.setRegistryName(RS.ID, "quartz_enriched_iron")
    }
}